_posts/2021-07-13-reading-group.md

```diff
@@ -1,10 +1,15 @@
 ---
 layout: post
 title: 2021 Reading Group
-date: 2021-07-14 11:00:00-0800
+date: 2021-09-09 09:00:00-0800
 description: Paper list and comments for reading group
 comments: false
 ---
 
-### 7.14 Paper list
-- [espnet](https://arxiv.org/abs/1804.00015)
+### 2021.09.07 Paper list
+- [Speech SimCLR: Combining Contrastive and Reconstruction Objective for Self-Supervised Speech Representation Learning](https://www.isca-speech.org/archive/interspeech_2021/jiang21_interspeech.html)
+- [Stochastic Attention Head Removal: A Simple and Effective Method for Improving Transformer Based ASR Models](https://www.isca-speech.org/archive/interspeech_2021/zhang21p_interspeech.html)
+- [Graph-PIT: Generalized Permutation Invariant Training for Continuous Separation of Arbitrary Numbers of Speakers](https://www.isca-speech.org/archive/interspeech_2021/neumann21_interspeech.html)
+
+### 2021.07.14 Paper list
```

Comment:
You can remove 0714's paper. It's just a test

Reply:
Removed it. Done.